crates/rspack_plugin_javascript/src/visitors/dependency/harmony_import_dependency_scanner.rs

```diff
@@ -4,18 +4,13 @@ use rspack_core::{
   ConstDependency, DependencyType, SpanExt,
 };
 use rustc_hash::{FxHashMap as HashMap, FxHashSet as HashSet};
-use swc_core::{
-  common::Span,
-  ecma::{
-    ast::{
-      AssignExpr, AssignOp, Callee, ExportAll, ExportSpecifier, Expr, Id, Ident, ImportDecl,
-      ImportSpecifier, Lit, MemberExpr, MemberProp, ModuleExportName, NamedExport, Pat, PatOrExpr,
-      Program, Prop,
-    },
-    atoms::JsWord,
-    visit::{noop_visit_type, Visit, VisitWith},
-  },
-};
+use swc_core::atoms::JsWord;
+use swc_core::common::Span;
+use swc_core::ecma::ast::{AssignExpr, AssignOp, MemberExpr, MemberProp};
```

Comment:
Maybe we could merge these ast imports?

Reply:
This is intentional. I think using the split could makes the code more readable.

Reply:
I could understand you split `atoms`, `common`, but does it make any difference split `ast` into four parts?